nevergrad/optimization/optimizerlib.py

```diff
@@ -806,8 +818,11 @@ def __init__(
         num_optims: int = 2,
         num_vars: tp.Optional[tp.List[int]] = None,
         multivariate_optimizer: base.ConfiguredOptimizer = CMA,
-        monovariate_optimizer: base.ConfiguredOptimizer = RandomSearch
+        monovariate_optimizer: base.ConfiguredOptimizer = RandomSearch,
+        progressive: bool = False
     ) -> None:
+        if progressive:  # The progressive setting is typically for noisy optimization, hence we switch to a noisy optimizer.
+            multivariate_optimizer = OptimisticNoisyOnePlusOne
```

Comment:
I'm a bit afraid of this kind of change without it being explicit in the docstring

Reply:
Fixed, thx.